src/check/parse/IR.zig

```diff
@@ -54,21 +54,11 @@ pub fn regionIsMultiline(self: *IR, region: Region) bool {
 pub fn regionInfo(self: *IR, region: Region, line_starts: std.ArrayList(u32)) base.DiagnosticPosition {
     const start = self.tokens.resolve(region.start);
     const end = self.tokens.resolve(region.end);
+    const info = base.DiagnosticPosition.position(self.source, line_starts, start.start.offset, end.end.offset) catch {
```

Comment:
Calculating region has some builtin sense checks, and we catch those errors here and panic to help isolate any obvious bugs.